README.md

```diff
@@ -0,0 +1,12 @@
+# Observatorium Operator
+
+[![Build Status](https://circleci.com/gh/observatorium/operator.svg?style=svg)](https://circleci.com/gh/observatorium/operator)
+
+Operator deploying the Observatorium project.
+Currently, this includes:
+
+0. operator itself.
```

Comment:
I don't have a strong opinion here, but this makes the list zero-indexed, should we make it start with a 1?

Reply:
GH markdown handles it if I'm not mistaken.

Reply:
not exactly, this is the result of
```
0. x
1. y
2. z
```

0. x
1. y
2. z

See that it becomes zero-indexed